datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -402,25 +403,55 @@ def in_developer_mode(self):
         self._log_deprecation('in_developer_mode')
         return False
 
+    def find_typos_in_options(self, user_config, models_config, level):
+        # only import it when running in python 3
+        from jellyfish import jaro_winkler_similarity
+
+        user_config = user_config or {}
```

Comment:
In what cases would `user_config` be set to `{}`?

Reply:
It could be `None` if `instances` is an empty list: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py#L194